lib/resty/etcd/v3.lua

```diff
@@ -446,8 +519,10 @@ local function request_chunk(self, method, host, port, path, opts, timeout)
 
         if body.result.events then
             for _, event in ipairs(body.result.events) do
-                event.kv.value = decode_base64(event.kv.value)
-                event.kv.value = decode_json(event.kv.value)
+                if event.kv.value then   -- DELETE not have value
```

Comment:
we can do this at a separate PR, one PR only do one thing. It will make the PR smaller and easier to review.

Reply:
this is last pr